bench/src/test/scala/benchmark/rabenchmarks.scala

```diff
@@ -24,7 +23,7 @@ object ReactiveAsyncBenchmarks extends PerformanceTest.Microbenchmark {
   val size = Gen.single(s"$nrOfCells cells")(nrOfCells)
 
   /* lattice instance for cells */
-  implicit val naturalNumberLattice: Lattice[Int] = new NaturalNumberLattice
+  implicit val naturalNumberUpdater: Updater[Int] = new NaturalNumberUpdater
```

Comment:
This should stay as before, and instead the implicit conversion from lattice to updater should take care of it.